### PR TITLE
Explore: Make it possible to close left pane of split view

### LINKED ISSUE #15995: Explore: Allow to close left side in split view
**What would you like to be added**:

A close button for Explore's left side in the split view.

**Why is this needed**:

- the left part may no longer be needed and most of the interaction happens on the right, closing would allow for more space
- this would allow for a workflow of continuously splitting right to drill down deeper

### PR DESCRIPTION
**Pull request by dprokop**
fixes #15995

### CODE REVIEW

**Review by davkal — APPROVED** [review]
LGTM

File: public/app/features/explore/state/reducers.ts [hunk 1]
```diff
@@ -489,7 +489,16 @@ export const itemReducer = reducerFactory<ExploreItemState>({} as ExploreItemSta
 export const exploreReducer = (state = initialExploreState, action: HigherOrderAction): ExploreState => {
   switch (action.type) {
     case ActionTypes.SplitClose: {
```
Comment (hugohaggmark) [inline public/app/features/explore/state/reducers.ts]: I think you could delete the ActionTypes.SplitClose and use this instead
```suggestion
    case splitCloseAction.type: {
```

**Review by hugohaggmark — APPROVED** [review]
LGTM

File: public/app/features/explore/state/reducers.ts (hunk 1 above)
Comment (dprokop) [reply public/app/features/explore/state/reducers.ts]: 👍

**Pull request merged**